<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Betaalmiddel</class>
 <widget class="QDialog" name="Betaalmiddel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icons/icons/credit.png</normaloff>:/icons/icons/credit.png</iconset>
  </property>
  <widget class="QTableView" name="tvLijst">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSluiten">
   <property name="geometry">
    <rect>
     <x>577</x>
     <y>11</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/icons/door_in.png</normaloff>:/icons/icons/door_in.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lbID">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbID">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>5</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbID</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbTijdstip">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>5</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbTijdstip</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbBedrag">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>5</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tbBedrag</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxBetaalwijze">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>40</y>
     <width>316</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbTijdstip">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tijdstip:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBedrag">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>10</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Bedrag:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBetaalwijze">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>45</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Betaalwijze:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpslaan">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>10</y>
     <width>66</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/icons/save_close.png</normaloff>:/icons/icons/save_close.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbBetaal_id">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>40</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
